### PR TITLE
fix label properties setter (issue #2477)

### PR DESCRIPTION
**Pull request by alisterburt**
bugfix and test for #2477 - could move the test into the big properties test if others prefer

### CODE REVIEW

File: napari/layers/labels/_tests/test_labels.py [hunk 1]
```diff
@@ -258,6 +258,17 @@ def test_properties():
...
 
+def test_default_properties_assignment():
+    """Test that the default properties value can be assigned to properties
+    see https://github.com/napari/napari/issues/2477
+    """
+    np.random.seed(0)
+    data = np.random.randint(20, size=(10, 15))
+
+    layer = Labels(data)
+    layer.properties = layer.properties
```
Comment (sofroniewn) [inline napari/layers/labels/_tests/test_labels.py]: Is the default `layer.properties` an empty dict `{}`?

The test might be a little bit more readable if it was

```suggestion
    layer.properties = {}
    assert layer.properties == {}
```

**Review by sofroniewn — APPROVED** [review]
Fix looks good, I've made a further suggestion for the test though to make it a little more readable for someone coming new. Thanks!!

**Comment by alisterburt** [comment]
committed your changes, thanks for the review!

**Comment by sofroniewn** [comment]
Thanks!! Will merge now :-)

**Comment by kevinyamauchi** [comment]
Thanks for the fix, @alisterburt ! I just ran into this bug, so looking forward to having this in the next release!

**Pull request merged**